service/src/main/java/org/broadinstitute/listener/relay/inspectors/SetDateAccessedInspector.java

```diff
@@ -88,10 +102,49 @@ public boolean inspectWebSocketUpgradeRequest(
     return checkLastAccessDateAndCallServiceIfExpired(relayedHttpListenerRequest);
   }
 
+  /**
+   * Inspect the request, calling Leonardo to updateDateAccessed on our resource if the request
+   * represents an action from a user other than the Leonardo service account.
+   */
   @Override
   public boolean inspectRelayedHttpRequest(RelayedHttpListenerRequest relayedHttpListenerRequest) {
+    logger.warn("DATE ACCESSED LISTENER !!! {}", relayedHttpListenerRequest.getUri());
+    logger.warn(relayedHttpListenerRequest.getHeaders().toString());
+    if (isLeonardoServiceAccountUser(relayedHttpListenerRequest)) {
```

Comment:
I see the overall approach. However, I am wondering if relying on an email address from the claim to identify what requests to ignore is a bit brittle. As an alternative, what about relying on the host name of caller - with a list in the config of what look for? Or filter by a custom header (e.g.  leo-status-check) that Leo can set when making status checks.

Reply:
Done! @giventocode thank you for all your feedback :)